pyproject.toml

```diff
@@ -31,7 +31,7 @@ authors = [
 license = { text = "BSD-3-Clause" }
 requires-python = ">=3.9"
 dependencies = [
-    "numpy",
+    "numpy >=1.21",
     "pynvml >=11.4.1",
```

Comment:
`pynvml` isn't listed as a [run dependency](https://github.com/rapidsai/ucx-py/blob/6f7c1aec5889eb08e488a3c9e85cdf5ad25e1368/dependencies.yaml#L67-L72) in `dependencies.yaml`, but `ucx` is listed? We need to make sure this is using `rapids-dependency-file-generator`.

Reply:
Not sure what's going on there

The recipe [loads dependencies from `pyproject.toml`]( https://github.com/rapidsai/ucx-py/blob/6f7c1aec5889eb08e488a3c9e85cdf5ad25e1368/conda/recipes/ucx-py/meta.yaml#L36-L38 ) atm

Maybe there is a better way to make this more generally consistent

Reply:
I'm fixing this and will file a PR shortly. It may supersede this PR if that's okay with you.

Reply:
Opened this: https://github.com/rapidsai/ucx-py/pull/952

Still working on it but it's close to complete, I think.